src/main/resources/liquibase/harness/compatibility/foundational/update/h2/createTable.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="1" author="as" labels="foundational">
+        <sql>UPDATE PUBLIC.foundational_test_table SET test_1=50, test_2=49, test_3=48, test_4=47, test_5=46, test_6=45, test_7=44, test_8=43, test_9=42, test_10=41, test_11=40,
```

Comment:
same, plz add comment to this changelog why default one doesn't work

Reply:
++